### DISCUSSION TITLE
Individual product prices

### DISCUSSION THREAD

**Discussion by cryexx** [Blair2004/NexoPOS#1364] [Ideas]
Hey,

for our project, it is equally important to have a specific product category, where you can enter the price individually in the product. 
For example, the blanket can not be scanned, because it is not yet entered into the system. So you click on the category in the POS and can then set an individual price.

I hope you understand what I mean ;)

**Comment by Blair2004** [comment]
Hi,
Sorry not really. Do you mean creating a product from the POS ?

**Comment by cryexx** [comment]
Hello,

thank you for your quick feedback.

I would like to have in sale products with individual purchase price. 
I set the product in the backend. I can then click on it in POS. The POS then asks me for the purchase price which I type in manually.

We use this for a cashless payment system.
In our example, when a customer wants to add money to their wristband, I want to be able to click on cash or card payment in the POS. The POS then asks me what amount will be charged. I type it in -> pay -> done.

We would like to make the solution directly through the POS, as this can also be done by cashiers without having to go to the backend.

**Reply by Blair2004** [reply]
I see,
Sorry defining the purchase price while adding a product to the cart, is not a feature we want to merge to the core at the moment.

Note that while adding the product to the cart, NexoPOS uses the ourchase price defined during the procurement. 

It's therefore possible your module to inject a popup that ask for the purchase price and add it to the product itself on cart.

**Reply by cryexx** [reply]
ok thanks. i will see how i can implement it.
